stdlib/public/runtime/ProtocolConformance.cpp

```diff
@@ -1276,19 +1400,43 @@ swift_conformsToProtocolMaybeInstantiateSuperclasses(
         return {dyldCachedWitnessTable, false};
   }
 
+  auto debugLogResult = [&](bool found, const char *source) {
+    if (IsDebugLog()) {
+      auto typeName = swift_getTypeName(type, true);
+      const char *status = found ? "found" : "not found";
+      printf("Check confomance %.*s to %s: %s, source: %s\n",
```

Comment:
Let's log to stderr to avoid interfering with the normal output of a program when this is turned on, just in case.